i18n/locales/en.json

```
@@ -57,5 +57,9 @@
   "pages.index.features.openSource.title": "Open Source",
   "pages.index.features.openSource.description": "Node.js is open source and actively maintained by contributors all over the world",
   "pages.index.features.everywhere.title": "Everywhere",
-  "pages.index.features.everywhere.description": "Node.js has been adapted to work in a wide variety of places"
+  "pages.index.features.everywhere.description": "Node.js has been adapted to work in a wide variety of places",
+  "components.Downloads.downloadHeader.activeSection": "downloads",
```

```suggestion
  "components.downloads.downloadHeader.activeSection": "downloads",
```

Folder name should respect the `camelCase` terminology. Please **read** docs.